.test-infra/jenkins/CommonTestProperties.groovy

```diff
@@ -27,8 +27,9 @@ class CommonTestProperties {
     enum Runner {
         DATAFLOW("DataflowRunner"),
         SPARK("SparkRunner"),
-        FLINK("TestFlinkRunner"),
+        FLINK("FlinkRunner"),
```

Comment:
You might want to use `TestFlinkRunner` which has the advantage that pipelines shut down for finite sources.

Reply:
ok